[32mINFO    [0m juju.model:model.py:2972 Waiting for model:
  mongodb-k8s/0 [idle] active: Primary
  mongodb-k8s/1 [idle] active: 
  mongodb-k8s/2 [idle] active: 
[32mINFO    [0m juju.model:model.py:2972 Waiting for model:
  mongodb-k8s/0 [executing] active: Primary
  mongodb-k8s/1 [executing] active: 
  mongodb-k8s/2 [executing] active:
[32mINFO    [0m pytest_operator.plugin:plugin.py:834 Model status:

Model  Controller          Cloud/Region        Version  SLA          Timestamp
test   microk8s-localhost  microk8s/localhost  3.4.4    unsupported  02:22:27Z

App            Version  Status       Scale  Charm          Channel      Rev  Address         Exposed  Message
application             active           1  application                   0  10.152.183.56   no       
mongodb-k8s             maintenance      3  mongodb-k8s                   0  10.152.183.94   no       backup started/running, backup id:'2024-09-01T02:22:23Z'
s3-integrator           active           1  s3-integrator  latest/edge   41  10.152.183.210  no       

Unit              Workload     Agent  Address       Ports  Message
application/0*    active       idle   10.1.216.210         
mongodb-k8s/0*    maintenance  idle   10.1.216.205         backup started/running, backup id:'2024-09-01T02:22:23Z'
mongodb-k8s/1     active       idle   10.1.216.208         
mongodb-k8s/2     active       idle   10.1.216.207         
s3-integrator/0*  active       idle   10.1.216.209         

[32mINFO    [0m pytest_operator.plugin:plugin.py:840 Juju error logs:

unit-new-mongodb-aws-1: 02:05:43 ERROR unit.new-mongodb-aws/1.juju-log new-mongodb-aws/1.is_unit_in_replica_set PyMongoError=No replica set members match selector "Primary()", Timeout: 1.0s, Topology Description: <TopologyDescription id: 66d3cbf6153f4e2d35b7ea86, topology_type: ReplicaSetNoPrimary, servers: [<ServerDescription ('new-mongodb-aws-1.new-mongodb-aws-endpoints', 27017) server_type: RSGhost, rtt: 0.0005916239997532102>]>
unit-new-mongodb-gcp-1: 02:20:19 ERROR unit.new-mongodb-gcp/1.juju-log new-mongodb-gcp/1.is_unit_in_replica_set PyMongoError=No replica set members match selector "Primary()", Timeout: 1.0s, Topology Description: <TopologyDescription id: 66d3cf629213988fca17a015, topology_type: ReplicaSetNoPrimary, servers: [<ServerDescription ('new-mongodb-gcp-1.new-mongodb-gcp-endpoints', 27017) server_type: RSGhost, rtt: 0.0006901589999870339>]>

[32mINFO    [0m pytest_operator.plugin:plugin.py:904 Forgetting main...